### PR TITLE
do not abort obj create/delete if we hit a master that does not suppo…

### PR DESCRIPTION
**Pull request by gabemontero**
…rt /swaggerapi/oapi

@openshift/sig-developer-experience fyi

to account for removal of /swapperapi/oapi REST endpoint

### CODE REVIEW

File: src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java [hunk 1]
```diff
@@ -20,8 +20,16 @@
...
+        String json = null;
+        try {
+        	json = httpGet(chatty, listener, overrides, this.getApiURL(overrides)
                 + "/swaggerapi" + apiDomain + "/v1");
+        } catch (Throwable t) {
+        	if (chatty) {
+        		listener.getLogger().println(t.getMessage());
+        	}
+        }
+        return json;
```
Comment (bparees) [inline src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java:32]: caller can handle null?

Comment (gabemontero) [reply src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java:32]: yes

Comment (gabemontero) [reply src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java:32]: https://github.com/openshift/jenkins-plugin/blob/master/src/main/java/com/openshift/jenkins/plugins/pipeline/model/IOpenShiftApiObjHandler.java#L27-L30

**Comment by gabemontero** [comment]
hmm perhaps this seldom updated repo is not fully wired for merging PRs ... or there was a hiccup when the tests passed

merging manually

**Pull request merged**